packages/core/src/builder.class.ts

```diff
@@ -1027,27 +1026,38 @@ export class Builder {
     }
   }
 
+  private static convertFnToString({
+    input,
+    keysToConvert,
+  }: {
+    input: Input;
+    keysToConvert: string[];
+  }) {
+    for (const key of keysToConvert) {
+      if (input[key] && typeof input[key] === 'function') {
+        const fn = input[key];
+        input = {
+          ...input,
+          [key]: `return (${fn.toString()}).apply(this, arguments)`,
+          ...(input.subFields == null
```

Comment:
there might be a lint rule type coercion with double equeals, maybe we do this instead?
```
  ...(Array.isArray(input.subFields)
    ? {
        subFields: input.subFields.map((input) => {
          return this.convertFnToString({ keysToConvert, input });
        }),
      }
    : {}),
```